test/Treasury.test.ts

```diff
@@ -1,7 +1,7 @@
 import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
 import {
-  AccessControlDAO,
```

Comment:
I think you need to update the types by recompiling, these should be named "DAOAccessControl" throughout this file